public/_posts/_newsletters/Yearn-Finance-Newsletter-58/hi.md

```diff
@@ -0,0 +1,128 @@
+
+इस तिजोरी में प्रवेश करने के लिए, आपको आर्बिट्रम को मेटामास्क, ब्रिज वाया सिनैप्स, हॉप, या आधिकारिक आर्बिट्रम ब्रिज, [curve.fi](https://arbitrum.curve.fi/) पर ट्राईक्रिप्टो में तरलता जोड़ने की आवश्यकता होगी, और अंत में, [yearn.finance/vaults](http://yearn.finance/vaults) पर जमा करें।
+
+हम आपको [Discord](https://discord.gg/8rF374XkXy), [Twitter](http://twitter.com/iearnfinance) पर जुड़कर, शामिल होने, प्रतिक्रिया देने और यर्न  समुदाय में भाग लेने के लिए आमंत्रित करते हैं, या [github] (http://github.com/yearn)। आओ हमारे साथ निर्माण करें।
+
+# यर्न सुरक्षा समीक्
+
+![](./image3.jpg?w=1000&h=563)
+
+यर्न में, हम सुरक्षा को बहुत गंभीरता से लेते हैं और आशा करते हैं कि Defi में सर्वोत्तम जोखिम-समायोजित प्रतिफल प्रदान करेंगे। 
+
+ऑडिटर्स की एक स्वतंत्र टीम, डेफीसेफ्टी की बदौलत, हमें ९३% का स्कोर मिला है। हमारे पहले से ही मजबूत सुरक्षा उपायों और संपूर्ण दस्तावेज़ीकरण के अलावा, यर्न का पूर्ण औपचारिक सत्यापन हो रहा है, जो ऑडिटिंग का एक अलग चरण है।
+
+पूरा ऑडिट पढ़ें [यहां](https://www.defisafety.com/pqrs/354).
+
+# २०२० विलय को स्पष्ट करना
+
+२०२०  के अंत में, यर्न फाइनेंस ने प्रमुख DEFI परियोजनाओं को एक साथ लाने के लक्ष्य के साथ कई "विलय" की घोषणा की, जिससे महत्वपूर्ण बुनियादी ढांचे पर केंद्रित एक पारिस्थितिकी तंत्र का निर्माण हुआ: एएमएम (सुशीस्वैप), लेंडिंग (क्रीम फाइनेंस), संस्थागत समर्थन (एक्रोपोलिस), और यील्ड मैनेजमेंट (अचार)। वित्त)।
+
+विभिन्न तरीकों से सफल होने के बावजूद, हम कभी भी एक वास्तविक विलय वाली इकाई नहीं बन पाए। आगे बढ़ते हुए, प्रत्येक प्रोटोकॉल स्वतंत्र रूप से काम करेगा जबकि मजबूत सहयोगी और यर्न और एक दूसरे के साथ अच्छे दोस्त बने रहेंगे।
+
+इस प्रकार, आयरन बैंक यर्न से स्वतंत्र एक परियोजना के रूप में कार्य करेगा।
+
+पूरा लेख [यहाँ](https://medium.com/iearn/clarifying-2020-mergers-an-independent-iron-bank-a6f8f3f4c25e)जाँचें
+
+# यर्न मेट्रिक्स में गोता लगाना
+
+![](./image4.png?w=1400&h=625)
+
+क्रिप्टो के प्रमुख लोकाचारों में से एक है, अपना स्वयं का शोध करना है, और यहां यर्न में, हम इस बात पर जोर देने की उम्मीद करते हैं कि प्रोटोकॉल के साथ बातचीत में उपयोगकर्ताओं को सावधानी बरतने में मदद करने के लिए पारदर्शी मेट्रिक्स हमारी सबसे बड़ी ताकत है।
+
+आमतौर पर इस्तेमाल की जाने वाली तीन साइटें हैं जिनकी हम अनुशंसा करते हैं [द वॉल्ट्स एट ईयर](https://vaults.yearn.finance/), [यर्न विजन](https://yearn.vision/), और [इयर वॉच](https://yearn.watch/).
+
+यर्न साइट पर वाल्ट सबसे सरल है, जो वॉल्ट रणनीतियों और यर्न में वाल्टों के प्रकारों का संक्षिप्त विवरण देता है। यर्न विज़न हमारा सबसे पूर्ण डैशबोर्ड है जो वॉल्ट के स्वास्थ्य, कुल मूल्य लॉक और अन्य एनालिटिक्स के बारे में जानकारी देता है। अंत में, यर्न वॉच मुख्य रूप से रणनीतिकारों के लिए कुछ रणनीतियों के स्वास्थ्य के बारे में जानकारी को समझने का एक उपकरण है।
+
+मार्कोवर्म्स और डार्क घोस्टी से यह पूरा लेख देखें [यहां](https://medium.com/iearn/diving-into-yearn-metrics-8c3fb0520927).
+
+आप इन साइटों पर सूचीबद्ध वाल्टों तक पहुंच सकते हैं और आज से [yearn.finance/#/vaults] पर कमाई शुरू कर सकते हैं।(https://yearn.finance/#/vaults).
```

Comment:
please fix the link here. the format should stay  [link _text](link_url) and there should not be any space or word between the brackets [] and the parentheses ().